### ISSUE TITLE
popup doesn't work with jquery 2.2.0 and jmq 1.4.5

### ISSUE THREAD

**Issue by suewonjp** [jquery/jquery-mobile#8381]
simple popup invocation doesn't work for the combination of the jquery 2.2.0 and jquery mobile 1.4.5 (both are recent versions)

Here is a very simple html file, which tries to only open a popup. It fails at least on my system.

[test7.html.zip](https://github.com/jquery/jquery-mobile/files/129610/test7.html.zip)

The error message is:   

> TypeError:  a.preventDefault is not a function. (In 'a.preventDefault()', 'a.preventDefault' is undefined)

But, other combinations work;  for instance:
- jq 1.9.1 and jqm 1.4.5
- jq  2.2.0 and jqm 1.3.2 

The bottom line is something seems wrong if both of the two are latest.

tested system:
- Safari - Version 9.0.3 (9537.86.4.4)
- OS X 10.9.5

**Comment by jacekkarczmarczyk** [comment]
https://github.com/jquery/jquery-mobile/blob/master/js/navigation/navigator.js#L135

```
				popstateEvent.originalEvent = {
					type: "popstate",
					state: null
				};
```

Then jquery.js tries to execute originalEvent.preventDefault which is not defined. Adding en empty method seems to work fine at least for this case. However I'm not sure whether this should be handled by jquery itself or should be fixed in jquery mobile.

**Comment by Gemorroj** [comment]
Not work if `data-history="true"` (http://api.jquerymobile.com/popup/#option-history).
Example code: http://js.do/code/84344
Problem affected in jquery version 2.2.0 and 2.2.1 (work fine in version 2.1.4).

**Comment by arschmitz** [comment]
jQuery 2.2 was released after jQuery Mobile 1.4.5 and is not supported by it. This is fixed in the latest code which will be for 1.5.0.

**Comment by jaspermdegroot** [comment]
Closing this ticket, based on @arschmitz his comment.

**Comment by arschmitz** [comment]
@jaspermdegroot i'm going to open this to discuss a possible 1.4.6 just to add compatibility at this weeks meeting we can update once a decision is made.

**Comment by Gemorroj** [comment]
@arschmitz any information about 1.4.6?

**Comment by apsdehal** [comment]
@arschmitz What is the final stance on 1.4.6?

**Comment by arschmitz** [comment]
We are planning a 1.4.6 once we get 1.5 out

**Comment by bassena** [comment]
How far are we away from jQM 1.5?

**Comment by apsdehal** [comment]
@bassena We are very close. We are working API docs and upgrade guide at the moment.

**Comment by bassena** [comment]
Glad to hear!

**Comment by tmarplatt** [comment]
While we wait for 1.5 (or 1.4.6), is there a known workaround for this issue? Other than downgrading jQuery, that is.

**Comment by Chez** [comment]
Still no news? For developers that need more than 1.4.5 offers, we are in limbo - drop jqm or wait and see, either way it could cost us dearly.

**Comment by arschmitz** [comment]
1.5-alpha1 is complete and we are just working on the accompanying documentation. If you want to try this out you can get the `git` version from our CDN or use the download builder on the master branch.

**Comment by Chez** [comment]
Ok, that is good news. One question, does 1.5 have right to left (rtl) ability? We use a customized version of 1.4.5 js and css files for our rtl version.

**Comment by jerroydmoore** [comment]
+1 this issue. The latest stable version of jqm should work with the latest version of jquery. Or better version pairing guidance should be given to developers, otherwise, the product appears unstable.
